### ISSUE TITLE
Create documentation for all of the kubejs binded functions

### ISSUE THREAD

**Issue by jasperalani** [game-design-driven/ptdye-plus#9]
after https://github.com/jasperalani/ptdye-plus/pull/4 has been merged:
* `src/main/java/com/kikis/ptdyeplus/kubejs/bindings/CellUtilJS.java`
* `src/main/java/com/kikis/ptdyeplus/kubejs/bindings/KeyUtilJS.java`
* and possibly this directory `src/main/java/com/kikis/ptdyeplus/appeng`

**Comment by CoolandonRS** [comment]
Honestly `com.kikis.ptdyeplus.appeng` should just be excluded entirely from kubeJS access (and if not, probably only `CellUtil`. It's really only intended to be called via `CellUtilJS`

**Comment by jasperalani** [comment]
Yeah I deleted the [filter.deny](https://github.com/game-design-driven/ptdye-plus/commit/9c16f540081d9915f605ed6d045f2c54181ddb40#diff-8f042c0c8b9d5f796efc8b263e6af867715baaf39688e9031aaae3841845a17bL55) part because I don't fully understand it.
